<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinGameConfView</class>
 <widget class="QWidget" name="JoinGameConfView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Players</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>27</number>
      </property>
      <item row="0" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Player 1:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Player 2:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="PlayerConfView" name="joinPlayerConf" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="ColorButton" name="player1ColorButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>311</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayoutJoin">
     <item>
      <widget class="QTreeView" name="joinGamesTreeView">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="manageIndexServersButton">
         <property name="text">
          <string>Manage Index Servers...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="joinButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Join</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>PlayerConfView</class>
   <extends>QWidget</extends>
   <header>playerconfview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>JoinGameConfView</receiver>
   <slot>_refreshClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manageIndexServersButton</sender>
   <signal>clicked()</signal>
   <receiver>JoinGameConfView</receiver>
   <slot>_manageIndexServersClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joinButton</sender>
   <signal>clicked()</signal>
   <receiver>JoinGameConfView</receiver>
   <slot>_joinClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>JoinGameConfView</receiver>
   <slot>_update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_refreshClicked()</slot>
  <slot>_manageIndexServersClicked()</slot>
  <slot>_joinClicked()</slot>
  <slot>_update()</slot>
 </slots>
</ui>
